tests/tests/account-settings.js

```diff
@@ -89,3 +90,42 @@ module.exports["Test profile changes"] = function (browser) {
     .execute("window.Meteor.logout()")
     .end();
 };
+
+module.exports["Test passing profile data to test app"] = function (browser) {
+  browser
+    .loginDevAccount()
+    .click("a.introjs-skipbutton")
+    .waitForElementNotPresent("div.introjs-overlay", short_wait)
+    // Make input field visible in order to manipulate it in Firefox
+    .execute(function () {
+      document.querySelector("input[type=file]")
+              .setAttribute("style", ""); 
+      }, [])
+    .perform(function (client, done) {
+      client.setValue("input[type=file]", testappPath, () => {
+        done();
+      })
+    })
+    .execute(function () {
+      const testappFile = document.querySelector("input[type=file]").files[0];
+      uploadApp(testappFile);
+    }, [])
+    .waitForElementVisible('#confirmInstall', medium_wait)
+    .click('#confirmInstall')
+    .waitForElementVisible('button.action', medium_wait)
+    .click('button.action')
+    .pause(15000)
+    .grainFrame()
+    .executeAsync(function (done) {
+      done(Meteor.sandstormUser());
+    }, [], function (result) {
+      this
+        .assert.containsText('#id', result.value.id)
```

Comment:
Hmm, it looks to me like these assertions here are checking that the content displayed on the screen matches the meteor user data present in the iframe.

What we'd really like to do, though, is check that the info matches what we've set on the account settings page -- that is, we're testing that the profile is being passed correctly from Sandstorm.

So what we want to do is go to the account settings page, set a particular name, handle, etc., and then open the meteor test app and verify that the profile data there matches what we set.